renovate.json

```diff
@@ -7,7 +7,9 @@
   "automerge": false,
   "separateMajorMinor": false,
   "timezone": "America/Tijuana",
-  "labels": ["comp: build", "dependencies", "action: merge"],
+  "labels": ["comp: build", "renovate managed", "action: merge"],
+  "keepUpdatedLabel": "renovate managed",
+  "rebaseWhen": "conflicted",
```

Comment:
Note that sometimes multiple PRs land and those may interplay with renovate update PRs. Is there a way to rebase like every 2nd day?

Reply:
Unfortunately, as I read it at least, there is not any sort of time based triggering.  The options are `never`, `conflicted` or `behind-base-branch`.